RELEASE_NOTES.md

```diff
@@ -2,7 +2,9 @@
 
 ## in develop
 
-* ..
+* WDL: Stages (frags/blocks) reuse applets as well. Previously only tasks were reused. **Breaking**: can break the logic
+  of any App reuse for CWL (even tasks maybe are not reused), because `ApplicationCompiler` and `WorkflowCompiler` now
+  look at the `DocContents` for checksum, and `SourceCode` attribute is now ignored.
```

Comment:
Do we need a separate CWL ticket to test this?

Reply:
I'd fix that in a separate ticket - otherwise APPS-994 gets real big and with lots of comments

Reply:
That's reasonable. If you could add a note to `CWL only` https://github.com/dnanexus/dxCompiler#limitations about a limited functionality for applet reuse for CWL. Thanks!

Reply:
added in [ab5d21b](https://github.com/dnanexus/dxCompiler/pull/291/commits/ab5d21bf743f7c619179f0c47c15839b98f53ec6)